CMakeLists.txt

```diff
@@ -59,6 +75,10 @@ if (CLAP_JUCE_EXTENSIONS_ROOT)
           CLAP_ID "org.surge-synth-team.monique-monosynth")
 endif()
 
+if(JUCE_ASIO_SUPPORT)
+  target_compile_definitions(${PROJECT_NAME} PUBLIC JUCE_ASIO=1)
+endif()
```

Comment:
Doesn’t this need to come after the env vst check? That is the asio is set later in the file so this should appear after that block? Or that block should appear before here?

Reply:
yes :)

Reply:
I removed the asio change from this pr, doesn't make much sense to push this untested.
need to set up a windows first